packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -59,6 +60,22 @@ type OperationHandler = (data: {
   row: LoadedRow;
 }) => void;
 
+/**
+ * A set of options to configure the scroll to index behavior.
+ */
+export interface ScrollToOptions {
+  /**
+   * Indicates if this operation will be animated. Defaults to `false`.
+   */
+  animate?: boolean;
```

Comment:
Should this be of type `Animation` instead of a boolean type? That way you can include ms to indicate the speed of the scroll rate.

Reply:
Unfortunately the browser doesn't give you any control over this using the native [`Element.scrollTo`](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo) behavior.